### ISSUE TITLE
VR Support?

### ISSUE THREAD

**Issue by BiatuAutMiahn** [wizgrav/clubber#2]
Would it be possible to implement VR support in the demo?

Thank you

**Comment by wizgrav** [comment]
@BiatuAutMiahn  Yes, it is possible but not completely straightforward. Clubber toy/tool fetch shaders from shadertoy which has some vr support but it needs some work by the shader writer to enable it so just a few support that for now. For 2D shaders it doesn't even make sense. You can check examples in shadertoy under the label "VR". I plan to implement most shadertoy features besides multipass, in the coming days.

If you are interested in VR development, in general, you can also use the clubber lib in a three.js or aframe project very easily. There are some examples in the readme you can check. In this case, the tool would still be useful for doing music analysis and designing your modulators. Translating the glsl snippets in three.js vector operations is very straightforward

**Comment by BiatuAutMiahn** [comment]
Ah ok, ty fro the feedback
